src/dev/custom-webpack.config.js

```diff
@@ -17,7 +15,11 @@ module.exports = config => {
    */
   config.module.rules.push({
     test: /\.js$/,
-    include: [path.resolve(__dirname, 'node_modules/lit-element'), path.resolve(__dirname, 'node_modules/lit-html')],
+    include: [
+      path.resolve('node_modules/lit-element'),
+      path.resolve('node_modules/lit-html'),
+      // path.resolve('dist/clr-core')
```

Comment:
Do we want to keep this here and commented out?

Reply:
I left it as more of a reminder we will need it for final testing of the prod builds.